src/integrationCapture.ts

```diff
@@ -160,9 +161,15 @@ export default class IntegrationCapture {
      * Captures Integration Ids from cookies and query params and stores them in clickIds object
      */
     public capture(): void {
-        const queryParams = this.captureQueryParams() || {};
-        const cookies = this.captureCookies() || {};
-        const localStorage = this.captureLocalStorage() || {};
+        let queryParams = this.captureQueryParams() || {};
+        let cookies = this.captureCookies() || {};
+        let localStorage = this.captureLocalStorage() || {};
+
+        if (this.captureMode === 'None' || !this.captureMode) {
```

Comment:
I would add the capture modes as exported constants to the src/constants.ts file under a new object called `IntegrationCaptureModes` or something like that.  This way they can be referenced from `mp-instance.ts` and `integrationCapture.ts`